pkg/github/github.go

```diff
@@ -686,6 +712,37 @@ func (g *GitHub) UploadReleaseAsset(
 	return asset, nil
 }
 
+// ToRequest builds an issue request from the set of options
+func (nio *NewIssueOptions) toRequest() *github.IssueRequest {
```

Comment:
I don't see that we use `Milestone` anywhere. Is that as expected?

Reply:
Milestones need an additional function to resolve their title to the numeric ID. Currently is ignored, it is coming in a followup PR.

Reply:
`github.GetMilestone()` followup PR:  https://github.com/kubernetes/release/pull/1965